### ISSUE TITLE
Systemd-boot kernel cmdline is being duplicated

### ISSUE THREAD

**Issue by ManuLinares** [dracutdevs/dracut#2174]
**Describe the bug**
Kernel cmdline is being duplicated, appends the systemd.machine_id=(machineid) every time you regenerate with "reinstall-kernels"

**Distribution used**
EndeavourOS, Arch.

**Dracut version**
056-1

**Init system**
systemd, systemd-boot

**To Reproduce**
1. Generate the systemd-boot entries with "reinstall-kernels"
2. Check "options" generated entry in "/boot/loader/entries/(machineid)-(kernelver).conf"
3. Last option "systemd.machine_id=(machineid)" is being duplicated

**Expected behavior**
Only one entry of "systemd.machine_id=(machineid)" should be left on systemd-boot loader entry.

**Additional context**
Added default cmdline with: **/etc/dracut.conf.d/cmdline.conf**

`
kernel_cmdline="root=PARTUUID=(partuuid) rw quiet rootflags=subvol=@ mitigations=off loglevel=3 nowatchdog nvme_load=YES nvidia-drm.modeset=1 udev.log_level=3 i915.fastboot=1 cpufreq.default_governor=schedutil
`

also, on the "journalctl" appears back to back multiple times, the same cmdline

```
systemd[1]: Starting dracut cmdline hook...
dracut-cmdline[219]: dracut-dracut-056
dracut-cmdline[219]: Using kernel command line parameters:  rd.driver.pre=btrfs root=PARTUUID=(partuuid) rw quiet rootflags=subvol=@ mitigations=off loglevel=3 nowatchdog nvme_load=YES nvidia-drm.modeset=1 udev.log_level=3 i915.fastboot=1 cpufreq.default_governor=schedutil   initrd=\(machineid)\6.1.8-1-cachyos\initrd root="PARTUUID=(partuuid)" rw quiet rootflags=subvol=@ mitigations=off loglevel=3 nowatchdog nvme_load=YES nvidia-drm.modeset=1 udev.log_level=3 i915.fastboot=1 cpufreq.default_governor=schedutil systemd.machine_id=(machineid) systemd.machine_id=(machineid) systemd.machine_id=(machineid) systemd.machine_id=(machineid) 
systemd[1]: Finished dracut cmdline hook.

```

**Comment by dalto8** [comment]
Yes, this has nothing to do with dracut.  It is probably an issue with https://gitlab.com/dalto.8/kernel-install-for-dracut
